src/app/Dice.ts

```diff
@@ -1,10 +1,20 @@
-import { GetRandomValue } from '../Interfaces/DiceInterface';
+import { RandomAnimalInterface } from '../Interfaces/RandomAnimalInterface';
 import { AnimalNames } from '../Enums/AnimalNamesEnum';
-import { sample } from 'lodash';
+import { sample, shuffle } from 'lodash';
+
+export class Dice implements RandomAnimalInterface {
+  private dice: AnimalNames[] = [];
+
+  addSide(animal: AnimalNames, probability: number): void {
+    if (probability === 0) {
+      return;
+    }
+    this.dice.push(animal);
+    this.addSide(animal, probability - 1);
```

Comment:
So the dice is created as it would be a real dice, with sides so that rabbit for exmaple is there 6 times, right?

Reply:
yes